sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/generate.ps1

```diff
@@ -1 +1 @@
-Start-AutoRestCodeGeneration -ResourceProvider "recoveryservicesbackup/resource-manager" -AutoRestVersion "v2"
\ No newline at end of file
+Start-AutoRestCodeGeneration -ConfigFileTag "package-2017-07"
```

Comment:
You can specify a tag but please keep the resourceprovider and autorestversion v2.
The latest version of autorest (v3) has a problem with csharp, so we are forcing all modules to generate with autorest v2.